db/seeds.rb

```diff
@@ -10,19 +10,23 @@
 
 apparel = Category.first.items.create([{name: "Hoody",
                       description: "Keeps you slightly warm",
-                      price: 80},
+                      price: 80,
+                      image: File.open('app/assets/images/hoody.jpg')},
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.